dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -1665,6 +1674,46 @@ protected synchronized void writeStreamOutput(SystemLogHolder holder, String txt
         holder.originalStream.println(txt);
     }
 
+    /**
+     * This method is accessed via reflection by the com.ibm.ws.logging.osgi.stackjoiner.bci.ThrowableInfo class
```

Comment:
if you change name of ThrowableInfo class, remember to change this comment too.

Reply:
fixed in [0113877](https://github.com/OpenLiberty/open-liberty/pull/16276/commits/011387765db34f7d2a9cf27d77487a12606c29ad)